tests/baselines/reference/classFieldSuperNotAccessibleJs.types

```diff
@@ -0,0 +1,85 @@
+//// [tests/cases/compiler/classFieldSuperNotAccessibleJs.ts] ////
+
+=== index.js ===
+// https://github.com/microsoft/TypeScript/issues/55884
+
+class YaddaBase {
+>YaddaBase : YaddaBase
+
+    constructor() {
+        this.roots = "hi";
+>this.roots = "hi" : "hi"
+>this.roots : any
+>this : this
+>roots : any
+>"hi" : "hi"
+
+        /** @type number */
+        this.justProp;
+>this.justProp : number
+>this : this
+>justProp : number
+
+        /** @type string */
+        this['literalElementAccess'];
+>this['literalElementAccess'] : any
+>this : this
+>'literalElementAccess' : "literalElementAccess"
+        
+        this.b()
+>this.b() : void
+>this.b : () => void
+>this : this
+>b : () => void
+    }
+    accessor b = () => {
+>b : () => void
+>() => {        this.foo = 10    } : () => void
+
+        this.foo = 10
+>this.foo = 10 : 10
+>this.foo : number | undefined
+>this : this
+>foo : number | undefined
+>10 : 10
+    }
+}
+
+class DerivedYadda extends YaddaBase {
+>DerivedYadda : DerivedYadda
+>YaddaBase : YaddaBase
+
+    get rootTests() {

```

Comment:
That is a separate issue - but I wonder... shouldn't such an invalid super expression result in an `errorType`? 🤔